QuantConnect.Polygon/PolygonDataDownloader.cs

```diff
@@ -13,10 +13,11 @@
  * limitations under the License.
  */
 
-using QuantConnect.Configuration;
+using NodaTime;
```

Comment:
Is this used?

Reply:
![image](https://github.com/QuantConnect/Lean.DataSource.Polygon/assets/45608740/2412681d-958a-4a20-aa17-7b97d4611d66)